op-challenger/game/fault/trace/select.go

```diff
@@ -0,0 +1,39 @@
+package trace
```

Comment:
bikeshed nit: This filename is a bit surprising. Was expecting something `access` related.

Reply:
Yeah good spot - it made more sense earlier in the refactoring process. :)